libs/designsystem/testing-base/src/lib/components/index.ts

```diff
@@ -0,0 +1,67 @@
+//TODO: This needs to be auto-generated!
```

Comment:
Actually, I think this should be a separate issue. 

What we want to do is to expose our mocks via a barrel file like this, because Ivy complains that these components are exposed through our TestingModule, without also being exported via a barrel file. 

We do this already for our KirbyModule, in https://github.com/kirbydesign/designsystem/blob/main/libs/designsystem/src/lib/index.ts (also hand-held), and something similar should happen here, but we should be able to easily automate it with our existing mock generation script.

Reply:
Agree!